src/test/java/org/opensearch/security/dlic/rest/api/AccountApiTest.java

```diff
@@ -25,26 +26,25 @@
 import org.junit.Assert;
 import org.junit.Test;
 import org.opensearch.security.test.helper.rest.RestHelper.HttpResponse;
-import java.util.Arrays;
 
 import static org.junit.Assert.*;
+import static org.opensearch.security.OpenSearchSecurityPlugin.LEGACY_OPENDISTRO_PREFIX;
+import static org.opensearch.security.OpenSearchSecurityPlugin.PLUGINS_PREFIX;
 
 @RunWith(Parameterized.class)
 public class AccountApiTest extends AbstractRestApiUnitTest {
-    private final String BASE_ENDPOINT;
```

Comment:
construct `ENDPOINT` using `BASE_ENDPOINT`, my comment was not to remove the `ENDPOINT` or `BASE_ENDPOINT` and hardcode `ENDPOINT + "/account"`, I referred that `ENDPOINT` can be constructed from `BASE_ENDPOINT` and it is only necessary to parametrize `BASE_ENDPOINT`.

Reply:
Fixed